hat/assets/js/apps/Iaso/domains/forms/hooks/useGetPossibleFields.ts

```diff
@@ -41,6 +53,40 @@ export const useGetPossibleFields = (formId?: number): Result => {
     return usePossibleFields(isFetchingForm, currentForm);
 };
 
+export const useAllPossibleFields = (
+    isFetchingForms: boolean,
+    allForms: Form[] = [],
+): AllResults => {
+    return useMemo(() => {
+        let allPossibleFields: PossibleFieldsForForm[] = [];
```

Comment:
```suggestion
        const allPossibleFields: PossibleFieldsForForm[] = [];
```
Pushing values into the array doesn't change its reference, so its safe (and preferred) to declare it with `const`